Parsers/ASimUserManagement/Parsers/vimUserManagementLinuxAuthpriv.yaml

```diff
@@ -0,0 +1,433 @@
+- Title: useradd
+  Link: https://manpages.ubuntu.com/manpages/lunar/en/man8/useradd.8.html
+- Title: userdel
+  Link: https://manpages.ubuntu.com/manpages/lunar/en/man8/userdel.8.html
+- Title: usermod
+  Link: https://manpages.ubuntu.com/manpages/lunar/en/man8/usermod.8.html
+Description: |
+  This ASIM parser supports normalizing Linux authpriv logs delivered using Syslog to the ASIM UserManagement normalized schema.
+ParserName: vimUserManagementLinuxAuthpriv
+EquivalentBuiltInParser: _Im_UserManagement_LinuxAuthpriv
+ParserParams:
+  - Name: starttime
+    Type: datetime
+    Default: datetime(null)
+  - Name: endtime
+    Type: datetime
+    Default: datetime(null)
+  - Name: srcipaddr_has_any_prefix
+    Type: dynamic
+    Default: dynamic([])
+  - Name: actorusername_has_any
+    Type: dynamic
+    Default: dynamic([])
+  - Name: targetusername_has_any
+    Type: dynamic
+    Default: dynamic([])
+  - Name: eventtype_in
+    Type: dynamic
+    Default: dynamic([])
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let parser = (
+        starttime:datetime=datetime(null), 
+        endtime:datetime=datetime(null),
+        srcipaddr_has_any_prefix: dynamic=dynamic([]), 
+        targetusername_has_any: dynamic=dynamic([]),
+        actorusername_has_any: dynamic=dynamic([]),
+        eventtype_in: dynamic=dynamic([]),
+        disabled:bool=false
+    ) {
+  let ActionLookup = datatable (Action:string, EventType:string)
+  [
+      "added",  "UserAddedToGroup",
+      "removed","UserRemovedFromGroup"
+  ];
+  let SeverityLookup = datatable (SeverityLevel:string, EventSeverity:string)
+  [
+      "info", "Informational",
+      "warn", "Low",
+      "err",  "Medium",
+      "crit", "High"
+  ];    
+  let ItemParser = (T:(SyslogMessage:string,SeverityLevel:string,ProcessID:int)) {
+      T
+      | lookup SeverityLookup on SeverityLevel
+      | extend ActingAppId = tostring(ProcessID)
+      | project-away SyslogMessage,SeverityLevel, ProcessID
+  };
+  let SyslogParsed = (
+      Syslog
+      | where not(disabled)
+      | where (isnull(starttime) or TimeGenerated >= starttime)
+           and (isnull(endtime) or TimeGenerated <= endtime)
+      | where (array_length(srcipaddr_has_any_prefix) == 0 or has_any_ipv4_prefix(HostIP, srcipaddr_has_any_prefix)) and
+                (array_length(targetusername_has_any) == 0 or (SyslogMessage has_any(targetusername_has_any))) and 
+                (array_length(actorusername_has_any) == 0 or (SyslogMessage has_any(actorusername_has_any)))
+      | where Computer in (_ASIM_GetSourceBySourceType('LinuxAuthpriv'))
+      | where Facility == "authpriv"
+          and ProcessName in ("useradd","usermod","userdel","groupadd","groupmod","groupdel","gpasswd")
+      | project-away EventTime,Facility,MG,CollectorHostName,SourceSystem,TenantId
+  );
+  union (
+      SyslogParsed
+      | where not(disabled)      
```

Comment:
@KustoKing we already applied this filter at line 87. Pls remove from other places.

Reply:
Done

Reply:
@KustoKing Please re-check. Change not visible